README.md

```diff
@@ -537,7 +538,7 @@ username of a user, group or channel, etc.).
 
 | Param | Type | Description |
 | --- | --- | --- |
-| chatId | <code>Number</code> &#124; <code>String</code> | Unique identifier for the target chat or username of the target supergroup or channel (in the format @channelusername) |
+| chatId | <code>Number</code> &#124; <code>String</code> | Unique identifier for the target chat or username of the target supergroup or channel |
```

Comment:
Me neither. It just happened when I "regenerated" readme, I decided to leave it as-is. I guess somebody before me forgot to regenerate readme after this minor change.